### PR TITLE
'What is Dogecoin' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Can someone please make sure the links in the article point to what will be the correct URLs? 🙏

### CODE REVIEW

File: content/dogepedia/articles/what-is-dogecoin.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
 [ author ]
   name = "Dogecoin"
 +++
  
-Let's talk about what Dogecoin really is. Is it its blockchain? Its community? Its future developments? Like all good cryptos, Dogecoin will need to be dynamic and reinvent itself, but always staying true to its origins of being the most fun crypto.
+Dogecoin is many things to many different people. At its heart, Dogecoin is the accidental crypto **movement** that makes people smile! 
 
-The Dogecoin Manifest can be a good source for this article.
\ No newline at end of file
+It is also an open-source peer-to-peer cryptocurrency that utilises [blockchain technology](https://www.dogecoin.com/dogepedia/articles/what-is-a-blockchain), a highly secure decentralised system of storing information as a public ledger that is maintained by a network of computers called [nodes](https://www.dogecoin.com/dogepedia/articles/what-is-a-node).
```
Comment (MishaBoar) [inline content/dogepedia/articles/what-is-dogecoin.md:10]:
```suggestion
It is also an open-source peer-to-peer cryptocurrency that utilises [blockchain technology](/dogepedia/articles/what-is-a-blockchain), a highly secure decentralised system of storing information as a public ledger that is maintained by a network of computers called [nodes](/dogepedia/articles/what-is-a-node). 

```

Comment (MishaBoar) [reply content/dogepedia/articles/what-is-dogecoin.md:10]: Relative paths

**Review by MishaBoar — APPROVED** [review]

File: content/dogepedia/articles/what-is-dogecoin.md (hunk 1 above)
Comment (MishaBoar) [reply content/dogepedia/articles/what-is-dogecoin.md:10]: I would recommend relative paths.

Comment (RichieJohn) [reply content/dogepedia/articles/what-is-dogecoin.md:10]: Great - thanks, MishaBoar! :)

**Pull request merged**